src/egon/data/datasets/power_plants/__init__.py

```diff
@@ -1142,7 +1250,10 @@ def fill_missing_bus_and_geom(gens, carrier):
         },
     )
 
-tasks = tasks + (wind_offshore.insert, assign_weather_data.weatherId_and_busId,)
+tasks = tasks + (
+    wind_offshore.insert,
+    assign_weather_data.weatherId_and_busId,
+)
 
 
```

Comment:
Please update the version number of the dataset

Reply:
Solved in 711eb374efcb28285a825f8b111dfa11169c8995